scripts/jenkins/ardana/gerrit/gerrit_handle_event.py

```diff
@@ -168,6 +168,8 @@ def handle_change_updated(change, dry_run=False):
                                     '' if direct else 'in',
                                     change.gerrit_url)
                           )
+            gerrit_review(ref_change, label='Code-Review', vote=0,
```

Comment:
You don't actually need a label/vote to post comments. In this case, it also doesn't make sense to have one, since the Gerrit Jenkins user doesn't actually use `Code-Review` labels.

Reply:
Right, good point. Thanks.